kimchi/src/proof.rs

```diff
@@ -201,6 +201,22 @@ impl<Evals> PointEvaluations<Evals> {
     }
 }
 
+impl<Evals: Copy> PointEvaluations<Evals> {
+    /// Mimic OCaml `point[0]`
+    ///
+    /// Short for `first`
+    pub fn fst(&self) -> Evals {
```

Comment:
We should change the Caml side instead. It is pretty ugly if we do point[0] in OCaml.

Reply:
See https://github.com/openmina/openmina/pull/890. I'll keep in mind to change in Pickles.